mozetl/topline/topline_summary.py

```diff
@@ -135,10 +135,16 @@ def search_aggregates(dataframe, attributes):
         .alias("count")
     )
 
+    source_whitelist = [
```

Comment:
Can we define the whitelist values in a single location shared by both topline and search rollups?